pkg/apis/controlplane/types.go

```diff
@@ -38,6 +38,8 @@ type PodReference struct {
 	Name string
 	// The namespace of this pod.
 	Namespace string
+	// The UID of this pod.
```

Comment:
Would you change "pod" to "Pods" in all comments?

Reply:
Sure